.gitignore

```diff
@@ -108,3 +108,6 @@ dmypy.json
 .pyre/
 
 # End of https://www.gitignore.io/api/python
+
+# virtual env kept in project dir
+venv
```

Comment:
Since I call all my venvs differently, they just all include the string `venv`, perhaps going with a regex would be better?
Not sure though.

Reply:
I keep them outside of the projects directories, so they have a clear separation :)
Ok, I get the regex argumentation, looks good to me 👍